packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -308,5 +308,6 @@ export const handler = async ({ provider, force }) => {
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
    console.error(c.error(e.message))
```